### ISSUE TITLE
Any suggestions on model configurations for waymo open dataset? 

### ISSUE THREAD

**Issue by andyzysraul** [open-mmlab/OpenPCDet#207]
Hello, Shaoshuai.  Thank you for your great work. 
I tried to adapt your code (PVRCNN) to waymo dataset (with data preprocessed similar to kitti format).
However, the results on the validation set were not good.  The recall_roi/rcnn_0.7 was 0.35/0.41.
My key configurations were : 
1. set pcloud range to [-80,-80,-3,80,80,1]; 
2. add an anchor paramed with [6.9, 2.6, 2.5];
3. increase num_filters for backnone_2d to [512,512] ;
4. change LR to 0.001, weight_decay to 0.005.
5. with Adam_onecycle optimizer
I guess the increase on diversity of amodal pointcloud distribution is a key factor,  but I could not find an efficient solution.
Could you give me some suggestions on model configurations for waymo dataset.？

PS: I'm a littile confused with the Adam_onecycle optimizer settings:1) as far as I know, 1-cycle LR policy needs a lower and an upper bounds which is absent in the PVRCNN yaml; 2) are LR_DECAY and DECAY_STEP_LIST needed for Adam one-cycle optimizer?
Looking forward to your kindly reply, thank you so much.

**Comment by sshaoshuai** [comment]
For training with waymo dataset, there are many configs that need to be tuned. Such as you could change the range of z-axis from [-3, 1] to [-1, 3] since the origin of waymo coordinate is on the ground. You may need to visualize the waymo point cloud to better understand the point cloud first. And we will also release the waymo related configs in the future but it will still take some time.

1. The upper bound is `LR`, the lower bound is `LR / DIV_FACTOR`.
2. Yes, one-cycle doesn't need `LR_DECAY` and `DECAY_STEP_LIST`.
